core/services/keeper/registry_synchronizer_sync.go

```diff
@@ -5,12 +5,16 @@ import (
 	"math/big"
 	"sync"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/pkg/errors"
 
 	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/ethkey"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+// KeeperList is a local lookup map that is refreshed every Registry sync.
+var KeeperList map[common.Address]int32
```

Comment:
won't this conflict if we run multiple keeper jobs? Maybe we should make it part of the `RegistrySynchronizer{}`

Reply:
fixed